store/generated_key/v2/eigenda.go

```diff
@@ -188,32 +188,52 @@ func (e Store) BackendType() common.BackendType {
 // Since v2 methods for fetching a payload are responsible for verifying the received bytes against the certificate,
 // this Verify method only needs to check the cert on chain. That is why the third parameter is ignored.
 func (e Store) Verify(ctx context.Context, versionedCert certs.VersionedCert, opts common.CertVerificationOpts) error {
-	switch versionedCert.Version {
-	case certs.V0VersionByte, certs.V1VersionByte:
-		e.log.Warn("EigenDA V2 certs are not supported anymore more for verification. Defaulting to successful verification.")
-		return nil
+	var retrievableDACert coretypes.RetrievableEigenDACert
```

Comment:
I think it would be cleaner if you just did `var rbn uint64` here, instead of keeping track of the `retrievableDACert`. The `rbn` would be set in each case below, just like you're currently doing for the `retrievableDACert`

Then there is only one "cert" variable, so you could rename `sumDACert` to `eigenDACert`

Reply:
tbh the cleanest solution would be updating the core interfaces to converge on a single one. if additional cert fields become necessary for verification interfaces then doing the declare -> conditional populate flow would scale poorly. will make the change